internal/config/auditsink.go

```diff
@@ -0,0 +1,83 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/spf13/viper"
+)
+
+var (
+	allowedVersions = []string{"v0.1"}
+)
+
+// AuditSinkConfig contains fields, which enable and configure
+// Flipt's various audit sink mechanisms.
+type AuditSinkConfig struct {
+	Version string       `json:"version" mapstructure:"version"`
```

Comment:
@markphelps Definitely. We should just provide them a version of the audit events, instead of letting them specify. I can make that change. Probably might require some more discussion between the three of us async.